src/main/scala/com/fulcrumgenomics/illumina/SampleSheet.scala

```diff
@@ -153,12 +153,14 @@ object SampleSheet {
 
   /** Validate that the sample ids are unique, and that the combination of sample name and library id are unique. */
   private def validateSamples(samples: Seq[Sample]): Unit = {
+    def sampleKey(s: Sample): String = s"${s.sampleName} ${s.libraryId}" + s.lane.map(" " + _).getOrElse("")
```

Comment:
If the lane is present on the sample, print it out on the error message.  Note that due to how this method is called in the `SampleSheet` constructor (183-186), only samples from the same lane are validated together.

Reply:
Is this used only for debugging?  If so a one-line scaladoc explaining that would be good.